api/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -3,6 +3,7 @@ syntax = "proto3";
 package envoy.extensions.filters.http.local_ratelimit.v3;
 
 import "envoy/config/core/v3/base.proto";
+import "envoy/config/route/v3/route_components.proto";
```

Comment:
I wonder if it would make more sense to move the `RateLimit` message to a different file (out of route_components.proto).
Unfortunately I think it must stay in "envoy/config/route/v3/" to avoid a breaking change.

Reply:
I think it make sense. But not just for RateLimit. There are lots of common base message here that shared by other components because historical reason. I will prefer to do that in an independent PR.